pkg/test/framework/components/istio/operator.go

```diff
@@ -645,6 +666,7 @@ func createRemoteSecret(ctx resource.Context, cluster resource.Cluster, cfg Conf
 		"x", "create-remote-secret",
 		"--name", cluster.Name(),
 		"--namespace", cfg.SystemNamespace,
+		"--service-account", constants.DefaultConfigServiceAccountName,
```

Comment:
Can we make this conditional on what kind of cluster? We don't want to run into a scenario where a (non-config) remote cluster passes in CI, but fails in an actual deployment due to permissions issues.

Reply:
We will create these serviceAccounts anyway if they do not exist as the PR in https://github.com/istio/istio/pull/28059

Reply:
This constant is `istiod-service-account`. By default this command sets things up to use `istio-reader`. If features are developed such that they require a permission that `istiod-service-account` has but `istio-reader` does not, integration tests will pass and users will fail since we don't document that they need to switch which service account to use.

Reply:
The issue is  istio-reader  lack one authority.  details see: https://github.com/istio/istio/pull/28389

after that PR is merged, we are safe to switch back to `istio-reader-service-accout` as the default one.

Reply:
My point is: we should use istio reader for remotes and istiod-... for config cluster.

Reply:
yes, after above PR is merged, we will use istio reader for remote.